Model/review.model.js

```diff
@@ -10,6 +10,19 @@ exports.selectReviewsById = (review_id) => {
         } else {
           return reviewOfIdNumber.rows[0];
         }
+      })
+      .then((reviewById) => {
+        const id = reviewById.review_id;
+        return db
+          .query(
+            "SELECT COUNT(*) FROM reviews LEFT JOIN comments ON reviews.review_id = comments.review_id WHERE reviews.review_id = $1;",
+            [id]
+          )
+          .then((results) => {
+            const countNumber = parseInt(results.rows[0].count);
+            reviewById.comment_count = countNumber;
+            return reviewById;
+          });
```

Comment:
I was more alluding to use doing this whole model with a single query, we can actually combine our query to select all the columns from the reviews table, and then ALSO use our COUNT() in the same query! 
When we use SELECT we can give that as many arguements as we want like

```SELECT reviews.review_id, reviews.votes, COUNT() AS comment_count etc ```